src/Components/App/App.js

```diff
@@ -51,6 +52,7 @@ const App = () => {
           return <JournalEntry journal={journal}/>
         }}/>
         <Route exact path='/how-you-felt'><FeelingsLog logs={userLogs}/></Route>
+        <Route path='*'><NotFound /></Route>
```

Comment:
purrrf

Reply:
Perfect!

Reply:
Correct! Any URL that we don't have a path for will redirect here!